src/test/java/uk/gov/companieshouse/chdorderconsumer/kafka/ItemOrderedKafkaConsumerTest.java

```diff
@@ -0,0 +1,201 @@
+package uk.gov.companieshouse.chdorderconsumer.kafka;
+
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.kafka.config.KafkaListenerEndpointRegistry;
+import org.springframework.messaging.MessageHeaders;
+import uk.gov.companieshouse.chdorderconsumer.exception.RetryableErrorException;
+import uk.gov.companieshouse.kafka.exceptions.SerializationException;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.serialization.AvroSerializer;
+import uk.gov.companieshouse.kafka.serialization.SerializerFactory;
+import uk.gov.companieshouse.orders.OrderReceived;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class ItemOrderedKafkaConsumerTest {
+    private static final String ORDER_RECEIVED_URI = "/order/ORDER-12345";
```

Comment:
`ORDER_RECEIVED_URI` - we might as well make this a realistic URI here for the avoidance of doubt?

Reply:
Sorry, should be clearer about this. The actual order URIs begin `/orders/..`?

Reply:
you're right totally missed that.